src-v2/Not/Not/Extensions/DoubleExtension.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Not.Extensions;
+public static class DoubleExtension
+{
+    public static double FloorWholeNumberToTens(this double number)
+    {
+        return (double)Math.Floor((decimal)number / 10) * 10;
+    }
```

Comment:
This seems wrong on multiple accounts:
- Why would the method be named `FloorWholeNumber`? How do you floor an integer? An then accept double?
- Does it work correctly for values like `5.6`?

Reply:
The idea is to floor the Competition's TotalDistance i.e. 44.5 to 40 so it is displayed as a 40km bracket instead of the exact length.

Reply:
It isn't very obvious what this method is doing. Also I don't think this is a good candidate for a generic extension in Not. Not everything that's dealing with base types such as int, string etc should be in the Not project. A good question would be : how useful do you image this method to be in the average project? It seems to me that it's a piece of specific logic related our domain, so that's where I'd put it. Also keep in mind it can be used in other places than Start - for example the dashboard component and RanklistMenu are good candidates.

Regarding the method itself - both the naming and implementation are confusing. It shouldn't work with `double` or `decimal` as you are rounding away precision, not the other way around.

Reply:
Still standing